ftw/subsite/content/subsite.py

```diff
@@ -85,7 +85,32 @@
         name="FromEmail",
         widget=atapi.StringWidget(
             label=_(u'label_fromemail', default=u'Email Senderaddress'),
-            description=_(u'help_fromname', default=u'')))
+            description=_(u'help_fromname', default=u''))),
+
+    atapi.TextField(
+        name='webstatsAnalyticsSnippet',
+        schemata='subsite',
+        default_content_type='text/html',
+        allowable_content_types=('text/html',),
+        validators=('isTidyHtmlWithCleanup', ),
```

Comment:
No, we should not allow `script`-tags, as this would introduce a security problem.
I think we need to treat this HTML differently: we have to trust the user (as it is done in the default site configuration too).
I propose to protect the field with a custom permission, so that only managers can do that by default. This reduces the group of attackable users.